tui/form.go

```diff
@@ -164,5 +164,6 @@ func (f *Form) BuildContent() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("Could not marshal into JSON\nmapped input:%v\nerror:%s", c, err)
 	}
+	fmt.Println("CONTENT:\n" + string(j))
```

Comment:
It was debugging output - I have two different personal notes somewhere to remember to remove it and I forgot... whoops. Good catch!